libtenzir/builtins/formats/kv.cpp

```diff
@@ -384,3 +384,4 @@ class parse_kv : public function_plugin {
 
 TENZIR_REGISTER_PLUGIN(tenzir::plugins::kv::kv_plugin)
 TENZIR_REGISTER_PLUGIN(tenzir::plugins::kv::read_kv)
+TENZIR_REGISTER_PLUGIN(tenzir::plugins::kv::parse_kv)
```

Comment:
Nit: Please adhere to our commit message style and remove the dot at the end of the commit message title.